v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -20,9 +20,21 @@
 
 /** Interface used by the SourcedbToSpanner pipeline to accept user input. */
 public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
+  @TemplateParameter.Enum(
```

Comment:
[nit] Would adding this parameter at the end avoid a re-numbering other paramters?

I don't think the order number matters a lot when a user executes a curl or gcloud command to start the template, the parameters are always key-value pairs.

Reply:
I would like this to be the first choice a user has to make in the UI.

Is it not related to the numbering here?

Reply:
That's right. Given the huge list of parameters we need to run a migration, we probably would have users with curl, gcloud or terraform.
But it's ok to reorder too.